pkg/cedar/core/gen/models/system_detail.go

```diff
@@ -35,6 +35,16 @@ type SystemDetail struct {
 	// Example: CMSS
 	Acronym string `json:"acronym,omitempty"`
 
+	// ato effective date
+	// Example: 2021-10-13T00:00:00.000Z
+	// Format: date
+	AtoEffectiveDate strfmt.Date `json:"atoEffectiveDate,omitempty"`
+
+	// ato expiration date
+	// Example: 2021-10-13T00:00:00.000Z
+	// Format: date
+	AtoExpirationDate strfmt.Date `json:"atoExpirationDate,omitempty"`
+
```

Comment:
these are the two we are not using - should i wire them up? or just leave?

@ClayBenson94 @mynar7

Reply:
@ClayBenson94 @mynar7 please let me know! this would include gql + app code to wire them up, but they would not be used

Reply:
If its not too much effort to wire up, it might be useful for other places where we fetch ATO information!